src/frontend/apps/standalone_site/src/features/Authentication/components/Authenticator.tsx

```diff
@@ -16,6 +16,24 @@ import { validateChallenge } from '../api/validateChallenge';
 const TARGET_URL_STORAGE_KEY = 'redirect_uri';
 const QUERY_PARAMS_CHALLENGE_TOKEN_NAME = 'token';
 
+function isLocalStorageAvailable() {
```

Comment:
this is already existing in `lib_video/src/utils/localstorage.ts`. And lucky you, it is tested !